src/DssSpell.sol

```diff
@@ -106,223 +66,138 @@ contract DssSpellAction is DssAction {
+        // Executive Proxy Poll: https://vote.makerdao.com/polling/Qmc9fd3j#poll-detail
+        // Onboard rETH Poll: https://vote.makerdao.com/polling/QmeEV7ph#vote-breakdown (Inside Proxy Spell)
+        // DAI Interest Rate Strategy Poll: https://vote.makerdao.com/polling/QmWodV1J#poll-detail (Inside Proxy Spell)
+        // Forum: https://forum.makerdao.com/t/2023-05-24-spark-protocol-updates/20958
+        DssExecLib.setIlkAutoLineParameters("DIRECT-SPARK-DAI", /* line */ 20 * MILLION, /* gap */ 20 * MILLION, /* ttl */ 8 hours);
+        DssExecLib.authorize(SPARK_PROXY, DssExecLib.esm());
+        ACLManagerLike(SPARK_ACL_MANAGER).addPoolAdmin(SPARK_PROXY);
+        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
+        DssExecLib.setChangelogAddress("EXEC_PROXY_SPARK", SPARK_PROXY);
+
+        // --- Non-Scope Defined Parameter Adjustments ---
+        // Poll: https://vote.makerdao.com/polling/QmQXhS3Z#poll-detail
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-2-non-scope-defined-parameter-changes-may-2023/20981
+
+        // Increase rETH-A line to 50 million DAI
+        // Increase rETH-A gap to 5 million DAI
+        DssExecLib.setIlkAutoLineParameters("RETH-A", /* line */ 50 * MILLION, /* gap */ 5 * MILLION, /* ttl */ 8 hours);
+
+        // Increase rETH-A Stability Fee to 3.74%
+        DssExecLib.setIlkStabilityFee("RETH-A", THREE_PT_SEVEN_FOUR, true);
+
+        // Increase CRVV1ETHSTETH-A Stability Fee to 4.24%
+        // NOTE: disabled for goerli because the collateral is not on the chain
+        // DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", FOUR_PT_TWO_FOUR, true);
+
+        // Increase WBTC-A Stability Fee to 5.80%
+        DssExecLib.setIlkStabilityFee("WBTC-A", FIVE_PT_EIGHT, true);
+
+        // Increase WBTC-B Stability Fee to 6.30%
+        DssExecLib.setIlkStabilityFee("WBTC-B", SIX_PT_THREE, true);
+
+        // Increase WBTC-C Stability Fee to 5.55%
+        DssExecLib.setIlkStabilityFee("WBTC-C", FIVE_PT_FIVE_FIVE, true);
+
+        // --- RWA015 (BlockTower Andromeda) ---
+        // Poll: https://vote.makerdao.com/polling/QmbudkVR#poll-detail
+        // Forum links:
+        //   - https://forum.makerdao.com/t/mip90-liquid-aaa-structured-credit-money-market-fund/18428
+        //   - https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969
+        //   - https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974
+
+        // --- USDP PSM Debt Ceiling ---
+        // Poll: https://vote.makerdao.com/polling/QmQYSLHH#poll-detail
+        // Forum: https://forum.makerdao.com/t/reducing-psm-usdp-a-debt-ceiling/20980
+        // Set PSM-USDP-A Debt Ceiling to 0 and remove from autoline
+        (,,,line,) = vat.ilks("PSM-PAX-A");
+        DssExecLib.decreaseIlkDebtCeiling("PSM-PAX-A", line / RAD, /* decrease global ceiling */ true);
```

Comment:
Note that a global debt ceiling decrease (last param) of 500M while that debt still exists (i.e the PSM is still full) can potentially cause a problem where the global debt is bigger than the global DC (in that state no more debt can be taken in any vault).

See summary here - https://discord.com/channels/893112320329396265/897483518316265553/1097862208446287912

In this case, the reduction should still leave a ~200M buffer (while currently, it is ~700M) on top of the current + drawable debt, so not an immediate problem, but I think it is better not to do it (i.e set last parameter as false).